kobiton-rest-api/get-devices-info/README.md

```diff
@@ -0,0 +1,149 @@
+
+<!-- /TOC -->
+
+## A. Overview
+This guideline will provide instructions and sample cases to demonstrate the consuming of `/devices` endpoint in Kobiton REST API.
+
+> More information about Kobiton REST API `/devices` endpoint can be found at [Api documentation](https://api.kobiton.com/docs/#devices)
+
+## B. Preparations
+### 1. Prerequisites
+- cURL installed.
+  - `Windows`: Visit https://curl.haxx.se/windows/ for instructions on how to download and install cURL on Windows.
+  - `macOS`: Installed by default
+
+### 2. Getting Kobiton Username and API Key
+Kobiton Username and API key are required for authenticating with Kobiton API.
+
+> If you don't have a Kobiton account, visit https://portal.kobiton.com/register to create one.
+
+To get your Kobiton Username and API Key, follow instructions at `IV. Configure Test Script for Kobiton` section on [our blog](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/).
+
+### 3. Getting Kobiton Group ID
+If your account is in an organization and assigned to one or more groups, you will need to get the group ID.
+
+Follow instructions at [Kobiton documentation](https://docs.kobiton.com/organization-management/automation-for-groups/default-group-setting/) to get your group ID.
+
+## C. Samples
+
+In this section, we will provide sample use cases for `/devices` endpoint using bash commands with cURL.
+
+- When execute the commands provided in this section, remember to replace these variables with your values:
+  - `<YOUR_KOBITON_USERNAME>`: Your Kobiton Username.
+  - `<YOUR_KOBITON_API_KEY>`: Your Kobiton API Key.
+  - `<YOUR_GROUP_ID>`: Your group Id
+
+- Expected output for all samples in this section:
+```javascript
+{"privateDevices":[...],"favoriteDevices":[...],"cloudDevices":[...]}
+```
+
+### 1. Get all devices
+Basic command to get devices data from Kobiton REST API. This will get all devices and devices info that the current user can access. 
+
+**Smaple:**
+```bash
+curl -X GET 'https://api.kobiton.com/v1/devices' \
+  -H 'Accept: application/json' \
+  -u <YOUR_KOBITON_USERNAME>:<YOUR_KOBITON_API_KEY>
+```
+
+**For user in org and group**
+```bash
+curl -X GET 'https://api.kobiton.com/v1/devices?groupId=<YOUR_GROUP_ID>' \
+  -H 'Accept: application/json' \
+  -u <YOUR_KOBITON_USERNAME>:<YOUR_KOBITON_API_KEY>
+```
+
+### 2. Getting online devices
+Filter and get only online devices from all devices that current user can access. Adding param:
+- `isOnline`: Check if the device is online **(True/False)**
+
+**Sample:**
+```bash
+curl -X GET 'https://api.kobiton.com/v1/devices?isOnline=true' \
+  -H 'Accept: application/json' \
+  -u <YOUR_KOBITON_USERNAME>:<YOUR_KOBITON_API_KEY>
+```
+
+**For user in org and group**
+```bash
+curl -X GET 'https://api.kobiton.com/v1/devices?isOnline=true&groupId=<YOUR_GROUP_ID>' \
+  -H 'Accept: application/json' \
+  -u <YOUR_KOBITON_USERNAME>:<YOUR_KOBITON_API_KEY>
+```
+
```

Comment:
Add a note to show what happens when a user has a group but don't specify the groupID in the param? (Same for getSession and related section if required)